### PR TITLE
Expand make vars for cluster name

### PR DESCRIPTION
**Pull request by schoren**
This PR is to propose a fix to #77

### CODE REVIEW

**Comment by bazel-io** [comment]
Can one of the admins verify this patch?

File: k8s/object.bzl [hunk 1]
```diff
@@ -136,12 +136,14 @@ def _common_impl(ctx):
   files = [ctx.executable.resolver]
 
   cluster_arg = ctx.attr.cluster
+  cluster_arg = ctx.expand_make_variables("cluster", cluster_arg, {})
```
Comment (mattmoor) [inline k8s/object.bzl]: For the sake of completeness, can you add this for `namespace` and `image_chroot` as well?  (any anything else I might be forgetting).

**Comment by mattmoor** [comment]
FYI the process for this (since Travis won't give secrets to PRs from other repos) is that once reviewed I'll merge this into a branch and open a second PR to make sure we don't break the integration tests.

This is awesome, thanks again.

**Comment by schoren** [comment]
Just added `namespace` and `image_chroot`. Let me know if I missed something. I'm new with bazel and rules_k8s, so I might miss stuff.

**Comment by mattmoor** [comment]
@bazel-io Test this please

**Pull request merged**